clients/drcov/postprocess/drcov2lcov.cpp

```diff
@@ -878,24 +878,30 @@ read_file_header(const char *buf)
      */
     PRINT(4, "Reading version number\n");
     if (dr_sscanf(buf, "DRCOV VERSION: %u\n", &version) != 1) {
-        WARN(2, "Failed to read version number");
+        WARN(1, "Failed to read version number");
         return NULL;
     }
     if (version != DRCOV_VERSION) {
-        WARN(2, "Version mismatch: file version %d vs tool version %d\n", version,
-             DRCOV_VERSION);
-        return NULL;
+        if (version == 2) {
```

Comment:
Maybe add a named constant for '2', describing that drcov version (similar to the OFFLINE_FILE_VERSION* constants in drmemtrace/trace_entry.h.

Reply:
Done.